backend/.rubocop.yml

```diff
@@ -8,6 +8,7 @@ AllCops:
     - "config/application.rb"
     - "db/schema.rb"
     - "app/spec"
+    - "db/migrate/20210421203610_create_active_storage_tables.active_storage.rb"
```

Comment:
Can you instead do something like...

```
- "db/migration/*.active_storage.rb"
```

so that _any_ active storage auto-generated migrations are ignored?